worker/worker.go

```diff
@@ -1276,7 +1276,7 @@ func (w *worker) Handler() http.Handler {
 		"GET    /account/:hostkey": w.accountHandlerGET,
 		"GET    /id":               w.idHandlerGET,
 
-		"GET    /rhp/contracts/active": w.rhpActiveContractsHandlerGET,
```

Comment:
I'm on the fence on whether to update this... we are still in beta but it would be a little annoying.